pkg/reduce/reduce.go

```diff
@@ -76,6 +79,7 @@ func NewDataForward(ctx context.Context,
 		watermarkPublishers: watermarkPublishers,
 		windowingStrategy:   windowingStrategy,
 		log:                 logging.FromContext(ctx),
+		keyed:               vertex.Spec.UDF.GroupBy.Keyed,
```

Comment:
can we make it boolean and do the conversion in reduce?

Reply:
done